rollup.config.js

```diff
@@ -0,0 +1,11 @@
+export default {
+    input: './src/index.js',
+    output: [{
+        file: './dist/theoplayer-web-native-picture-in-picture.js',
+        format: 'umd',
+        name: 'THEOplayerNativePictureInPicture'
```

Comment:
If you load the library directly through a `<script>` tag instead of bundling it together with your app, then you currently have to use:
```javascript
const pipPlayer = new THEOplayerNativePictureInPicture.NativePictureInPictureTHEOplayer(THEOplayer, player, {defaultUI: true});
```
Feel free to pick a different name. 😛